data/zilliqa/assets/zil1jy3g5j9w5njqwxuuv3zwkz9syyueelmu7g080v/meta.json

```diff
@@ -3,14 +3,18 @@
   "symbol": "FEES",
   "web": "https://unifees.io",
   "supply": "1000000000",
+  "decimal": "4",
+  "publicTeam": true,
+  "holders": true,
   "links": {
     "telegram": "https://t.me/unifees",
     "linkedin": "https://www.linkedin.com/company/unifees",
     "twitter": "https://twitter.com/UnifeesToken",
     "github": "https://github.com/Unifees/Unifees-Token"
+    "whitepaper": "https://unifees.io/Unifees-Whitepaper.pdf"
```

Comment:
not at the right place

Reply:
ok got it, i'll edit this

Reply:
data/zilliqa/assets/zil1jy3g5j9w5njqwxuuv3zwkz9syyueelmu7g080v/meta.json

please check